app/src/main/java/org/schabi/newpipe/report/ErrorActivity.java

```diff
@@ -377,9 +461,18 @@ private String getUserActionString(final UserAction userAction) {
         }
     }
 
-    private String getContentLangString() {
-        String contentLanguage = PreferenceManager.getDefaultSharedPreferences(this)
+    private String getContentCountryString() {
+        String contentCountry = PreferenceManager.getDefaultSharedPreferences(this)
                 .getString(this.getString(R.string.content_country_key), "none");
+        if (contentCountry.equals(getString(R.string.default_localization_key))) {
+            contentCountry = Locale.getDefault().toString();
+        }
+        return contentCountry;
+    }
+
+    private String getContentLanguageString() {
```

Comment:
Same, don't duplicate, just use `Localization.getPreferredLocalization(this).getLocalizationCode();`

But remember that it's useless now. See https://github.com/TeamNewPipe/NewPipe/issues/3089#issuecomment-585832060. So maybe we should gather app language, with `Localization.getAppLocale(this).getLanguage();`

Reply:
Ah yes. I wrote that code back in October along with some other changes for more debug info regarding the parsed YouTube pages and didn't think of your recent changes when rebasing.

Reply:
I think `contentCountry` might be useful in the future as soon as we enabled support for it again. I corrected the code but didn't include the content country in the report and also added a field for the app language.

Reply:
Content country != Localization

Reply:
Done